internal/tiltfile/tiltfile_test.go

```diff
@@ -63,7 +63,7 @@ func newGitRepoFixture(t *testing.T) *gitRepoFixture {
 	}
 }
 
-func (f *gitRepoFixture) LoadManifests(names ...string) []model.Manifest {
+func (f *gitRepoFixture) LoadManifests(names ...string) ([]model.Manifest, model.YAMLManifest) {
```

Comment:
since this is only used for testing, i'm surprised you didn't keep the interface the same?

it seems weird to have a test function that has 2 return values, but all callers only care about 1.

Reply:
So far callers only care about one return value, but there are gonna be some that care about both... this is fair tho, i might split it out.